corporate/lib/analytics.py

```diff
@@ -147,3 +153,60 @@ def get_plan_data_by_remote_realm() -> Dict[int, Dict[int, RemoteActivityPlanDat
                 current_server_data[realm_id] = plan_data
 
     return remote_server_plan_data_by_realm
+
+
+def get_realm_user_counts(
```

Comment:
This should be `get_remote_realm_user_counts`, right?

Reply:
Renamed as suggested.